ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -46,6 +46,8 @@
 /* package */ class ReactViewBackgroundDrawable extends Drawable {
 
   private static final int DEFAULT_BORDER_COLOR = Color.BLACK;
```

Comment:
It's used in the subsequent lines, as I split out the rgb and alpha components. Hope that makes sense.